src/org/traccar/Context.java

```diff
@@ -137,10 +137,13 @@ public static EventForwarder getEventForwarder() {
     public static void init(String[] arguments) throws Exception {
 
         config = new Config();
-        if (arguments.length > 0) {
-            config.load(arguments[0]);
+        if (arguments.length <= 0) {
+            System.out.println("No configuration file given.");
+            System.exit(0);
```

Comment:
I don't like `System.exit` as well. Can we just throw an exception instead? That was we don't need log either.